schema/partner_categories.js

```diff
@@ -12,6 +13,9 @@ const PartnerCategories = {
     size: {
       type: GraphQLInt,
     },
+    category_type: {
+      type: GraphQLString,
+    },
```

Comment:
I think this should be an `EnumType` of possible values. Lemme know if you need halp setting that up